### PR TITLE
Add Perplexity AI

### PR DESCRIPTION
**Pull request by PierreMesure**
To celebrate OTA's 4 year anniversary 🎂, I'm adding a few services that I would like to see tracked. 😊

### CODE REVIEW

**Comment by PierreMesure** [comment]
There seems to be a few problems:
- the API terms and conditions have names that aren't accepted, is that an issue?
- the webpage doesn't want to be crawled. I tried a few user agents and additional headers with no avail, I always get a 403. What's a good way to proceed? Does OTA have support for a headless browser for these hard cases?

File: declarations/Perplexity.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Perplexity",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.perplexity.ai/hub/legal/terms-of-service",
+      "select": [
+        "[name=Post]"
+      ],
```
Comment (clementbiron) [inline declarations/Perplexity.json:8]: Well done for this selector!

File: declarations/Perplexity.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "executeClientScripts": true
+    },
+    "Developer Terms": {
+      "fetch": "https://www.perplexity.ai/hub/legal/perplexity-api-terms-of-service",
+      "select": [
+        "[name=Post]"
+      ],
+      "executeClientScripts": true
+    },
+    "Business Privacy Policy": {
```
Comment (clementbiron) [inline declarations/Perplexity.json:25]: In my opinion, this document is not a `Business Privacy Policy`. I think it's a document that specifies the treatment of personal data when users interact programmatically with the service. So we're maybe talking about a new type, which could be called, for example, `Developer Privacy Policy`. If you agree with my analysis, I suggest that you delete the tracking of `Business Privacy Policy` and I will open a [discussion](https://github.com/OpenTermsArchive/terms-types/discussions) to add this new type of terms.

**Review by clementbiron — COMMENTED** [review]
Thank you very much Pierre for this anniversary contribution, it's really nice!

> the API terms and conditions have names that aren't accepted, is that an issue?

No, this isn't a problem. Very often the names used vary from one service to another, which is why we unifying in Open Terms Archive with [terms-types](https://github.com/OpenTermsArchive/terms-types/tree/main) database.

> the webpage doesn't want to be crawled. I tried a few user agents and additional headers with no avail, I always get a 403. What's a good way to proceed? Does OTA have support for a headless browser for these hard cases?

Locally, I don't get a 403 error (do you ?), but in CI or on our test server there is indeed a [403 error](https://github.com/OpenTermsArchive/GenAI-declarations/actions/runs/9388390024/job/25853525070?pr=183) at the fetch. It may be an anti-bot protection or it may be linked to the way we retrieve documents. In any case, we won't be able to solve this problem now.

I've discussed this with @Ndpnt  and we think it's better to validate your PR even if we have a 403 error in CI because:
- if the engine is updated, the problem could correct itself
- if it's an anti-bot protection, it can stop from one day to the next
- the issue that will be open will shows that the service is already tracked, there's no need to add it

File: declarations/Perplexity.json (hunk 2 above)
Comment (PierreMesure) [reply declarations/Perplexity.json:25]: Agree that it fits better but "Developer privacy policy" sounds like it's about the privacy of the developers. Why not introduce "API privacy policy" and "API terms and conditions"? Surely that's a pretty common use case?

**Comment by PierreMesure** [comment]
It works for me from the web browser but I was never able to make it work programmatically or with a REST client like Postman. If a headless browser doesn't cut it, I'm not sure what will.

File: declarations/Perplexity.json (hunk 2 above)
Comment (clementbiron) [reply declarations/Perplexity.json:25]: The [discussion](https://github.com/OpenTermsArchive/terms-types/discussions/29) I've just opened will enable us to check that these are common cases. We integrate new types of terms when we have been able to reference 3 documents of the same type and we have found a consensus name.

**Review by clementbiron — APPROVED** [review]

**Comment by clementbiron** [comment]
🚀

**Pull request merged**